manifests/job/daily.pp

```diff
@@ -27,14 +27,14 @@
 # Requires:
 #
 # Sample Usage:
-#   cron::daily { 'mysql_backup':
+#   cron::job::daily { 'mysql_backup':
 #     minute      => '1',
 #     hour        => '3',
 #     environment => [ 'PATH="/usr/sbin:/usr/bin:/sbin:/bin"' ],
 #     command     => 'mysqldump -u root my_db >/backups/my_db.sql',
 #   }
 #
-define cron::daily (
+define cron::job::daily (
```

Comment:
This would be a breaking change for users who aren't using hiera but defining cron::daily resources directly in their manifests.